### PR TITLE
Eventlist UI

### PR DESCRIPTION
**Pull request by Lhyejin**
#67 
- Event Event Card Format
  - Add Time
  - Add Address
  - parsing address
  - Change date format
  - Title Capitalize
  - Description wrapping

- Tag Button Format
  - Disable Tag button that there's not events
  - Add Clicked Tag Button Style
  - View tag scroll slide

### CODE REVIEW

File: web/src/components/eventlist-filters.js [hunk 1]
```diff
@@ -55,13 +54,14 @@ class EventlistFilters extends Component {
      */
     super(props);
 
+
```
Comment (lsh9034) [inline web/src/components/eventlist-filters.js]: why did you add this line?

File: web/src/components/event-cards.js [hunk 2]
```diff
@@ -111,59 +112,96 @@ class EventCards extends Component {
...
+      scheduleIndex < this.props.events[cardIndex].schedule.length;
+      ++scheduleIndex){
+      scheduleDate = new Date(this.props.events[cardIndex].
+        schedule[scheduleIndex].startDateTime);
+      if(scheduleDate > nowDate){
+        return [scheduleIndex, <Datetime>{this.props.events[cardIndex].
+          schedule[scheduleIndex]}</Datetime>]
+      }
+    }
+    return [0, 'Ended']
```
Comment (ethkim) [inline web/src/components/event-cards.js]: This seems a little dangerous. What if schedule[] is empty? Then the code below (caller of FilterSchedule) will try to access 0th element of schedule.
